From a5435aabdeb4516f249db95b399d6d97f4084d31 Mon Sep 17 00:00:00 2001
From: Jordan Petridis <jordan@centricular.com>
Date: Mon, 5 Sep 2022 04:42:55 +0300
Subject: [PATCH 1/3] Revert "Merge pull request #9 from nanonyme/2.3.0"

This reverts commit 7fbabfb3ccd5170e0d02e4c84bbe3ff694ba23e3, reversing
changes made to dfe5e4ecc78c4b13b31b8f8c611373a769fe838c.
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 3a7dc05..6996583 100644
--- a/meson.build
+++ b/meson.build
@@ -9,7 +9,7 @@ project('noopenh264', ['cpp'],
 )
 
 major_version = '6'
-matching_version = '2.3.0'
+matching_version = '2.2.0'
 
 pkgconfig = import('pkgconfig')
 

From eee5ae5d4f0c841a057ddefb6c1e22c6b05f68bd Mon Sep 17 00:00:00 2001
From: Jordan Petridis <jordan@centricular.com>
Date: Mon, 5 Sep 2022 04:42:29 +0300
Subject: [PATCH 2/3] Revert "Bump package version to 2.3.0"

This reverts commit 750147f1e5af5cd9f47ffabef996c335b1ba0481.
---
 debian/changelog | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/debian/changelog b/debian/changelog
index 13c2c80..14e427b 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,9 +1,3 @@
-noopenh264 (2.3.0) eos; urgency=medium
-
-  * Update to match new upstream version.
-
- -- Will Thompson <wjt@endlessos.org>  Mon, 01 Aug 2022 18:07:27 +0100
-
 noopenh264 (2.1.1-2) eos; urgency=medium
 
   * Revert changes to add plumbing to openh264 flatpak runtime, this is

From d8c760615f2eaff6f592ae786b4b7f64a2b39c41 Mon Sep 17 00:00:00 2001
From: Jordan Petridis <jordan@centricular.com>
Date: Mon, 5 Sep 2022 04:43:35 +0300
Subject: [PATCH 3/3] codec_ver.h: Sync version header for 2.2.0

---
 codec/api/svc/codec_ver.h | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/codec/api/svc/codec_ver.h b/codec/api/svc/codec_ver.h
index f45ab3f..adc1c3e 100644
--- a/codec/api/svc/codec_ver.h
+++ b/codec/api/svc/codec_ver.h
@@ -4,12 +4,12 @@
 
 #include "codec_app_def.h"
 
-static const OpenH264Version g_stCodecVersion  = {2, 1, 1, 2005};
-static const char* const g_strCodecVer  = "OpenH264 version:2.1.1.2005";
+static const OpenH264Version g_stCodecVersion  = {2, 2, 0, 2201};
+static const char* const g_strCodecVer  = "OpenH264 version:2.2.0.2201";
 
 #define OPENH264_MAJOR (2)
-#define OPENH264_MINOR (1)
-#define OPENH264_REVISION (1)
-#define OPENH264_RESERVED (2005)
+#define OPENH264_MINOR (2)
+#define OPENH264_REVISION (0)
+#define OPENH264_RESERVED (2201)
 
 #endif  // CODEC_VER_H
